shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2615,11 +2627,12 @@ void WebContents::Print(gin::Arguments* args) {
     settings.SetIntKey(printing::kSettingDpiVertical, dpi);
   }
 
+  std::string error;
   print_task_runner_->PostTaskAndReplyWithResult(
-      FROM_HERE, base::BindOnce(&GetDefaultPrinterAsync),
+      FROM_HERE, base::BindOnce(&GetDefaultPrinterAsync, &error),
```

Comment:
Since we are already using `PostTaskAndReplyWithResult`, `GetDefaultPrinterAsync` can just return both the result and error, ex: `std::pair`.

Reply:
done!